plugins/woocommerce-admin/client/index.js

```diff
@@ -96,6 +96,7 @@ if ( appRoot ) {
 
 	const wrap =
 		wpBody.querySelector( '.wrap.woocommerce' ) ||
+		wpBody.querySelector( '.woocommerce' ) ||
```

Comment:
I think this is a bit too general as it will pick up any element with class woocommerce, which is probably many things -

What do you think about something more specific like `div.woocommerce > div.wrap` ?

Reply:
I agree, example is this element using `.woocommerce`

<img width="743" alt="image" src="https://user-images.githubusercontent.com/3747241/177069643-a2a21169-6d91-4883-90d7-e4ab165cf534.png">

Reply:
I'll make the changes